mass_editing/wizard/mass_editing_wizard.py

```diff
@@ -268,4 +268,12 @@ def read(self, fields, load='_classic_read'):
         if fields:
             # We remove fields which are not in _fields
             real_fields = [x for x in fields if x in self._fields]
-        return super(MassEditingWizard, self).read(real_fields, load=load)
+        res = super(MassEditingWizard, self).read(real_fields, load=load)
+        # In order to avoid null value in client side we set dummy values fo
+        # dynamic fields build by fields_view_get
+        for item in res:
+            for field in fields:
```

Comment:
You can save a lot of time (performance) avoiding a loop-of-loop for a same result.
You can use `set` too.

 I mean,
Improvement 1.0 avoiding loop-of-loop
```python
...
res = super(MassEditingWizard, self).read(real_fields, load=load)
dummy_fields  = []
for field in fields:
    if field not in self._fields:
        dummy_fields.append(field)
dummy_fields_values = dict.fromkeys(dummy_fields)
for item in res:
    item.update(dummy_fields_values)
```


Improvement 2.0 using `set`
```python
...
res = super(MassEditingWizard, self).read(real_fields, load=load)
dummy_fields = dict.fromkeys(set(fields) - set(self._fields))
for item in res:
    item.update(dummy_fields)
```

The performance is so faster in the last one vs the current one and the code is so clear.